server/protocols/protocols-pop3/src/main/java/org/apache/james/pop3server/mailbox/MailboxAdapter.java

```diff
@@ -43,47 +42,10 @@
 import com.google.common.collect.ImmutableList;
 
 public class MailboxAdapter implements Mailbox {
-
-    private abstract static class POP3FetchGroup implements FetchGroup {
-        @Override
-        public Set<PartContentDescriptor> getPartContentDescriptors() {
-            return new HashSet<>();
-        }
-    }
-
-    private static final FetchGroup FULL_GROUP = new POP3FetchGroup() {
-
-        @Override
-        public int content() {
-            return BODY_CONTENT | HEADERS;

```

Comment:
can you confirm `BODY_CONTENT | HEADERS` = `FetchGroup.FULL_CONTENT`?

Reply:
Logically speaking yes!